Natsort import error

# Brief Description

The module `natsort` isn't found because it was added (in #627) to dev requirements but needs to be in the main requirements file. It is imported with all functions from the init script. 

Rather than requiring it, perhaps it could also be brought in with a `try`, `except` per #97 

# Error Messages

```
 /usr/local/lib/python3.7/site-packages/janitor/functions.py:25: in <module>
     from natsort import index_natsorted, natsorted
 E   ModuleNotFoundError: No module named 'natsort'
```

--- a/.bumpversion.cfg
+++ b/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.20.1
+current_version = 0.20.2
 commit = True
 tag = True
 

--- a/janitor/__init__.py
+++ b/janitor/__init__.py
@@ -19,4 +19,4 @@ def get_features_targets(*args, **kwargs):
     return _get_features_targets(*args, **kwargs)
 
 
-__version__ = "0.20.1"
+__version__ = "0.20.2"

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,3 +4,4 @@ scikit-learn>=0.19.1
 xlrd>=0.9.0
 pandas-flavor==0.1.2
 requests
+natsort

--- a/setup.py
+++ b/setup.py
@@ -46,7 +46,7 @@ extra_all = extra_biology + extra_engineering + extra_spark
 
 setup(
     name="pyjanitor",
-    version="0.20.1",
+    version="0.20.2",
     description="Tools for cleaning pandas DataFrames",
     author="Eric J. Ma",
     author_email="ericmajinglong@gmail.com",

